### ISSUE TITLE
Images for 

### ISSUE THREAD

**Issue by developer0hye** [developer0hye/Setup-for-Imagenet#1]
![default](https://user-images.githubusercontent.com/35001605/52109318-6d002800-2640-11e9-8138-c478c94da897.png)

![1](https://user-images.githubusercontent.com/35001605/52569830-626e3b80-2e55-11e9-91a2-16ac0684a89a.png)

![densenet](https://user-images.githubusercontent.com/35001605/52570579-11f7dd80-2e57-11e9-9fb0-c941a04d611e.png)
![resnet](https://user-images.githubusercontent.com/35001605/52113479-b787a180-264c-11e9-9024-5bd380e18560.png)
![vgg](https://user-images.githubusercontent.com/35001605/52113480-b787a180-264c-11e9-9741-b117edf52839.png)

![densenet](https://user-images.githubusercontent.com/35001605/52570730-68651c00-2e57-11e9-95da-030039ab3263.png)
![resnet](https://user-images.githubusercontent.com/35001605/52570731-68fdb280-2e57-11e9-9c34-ba0646161edd.png)
![vgg](https://user-images.githubusercontent.com/35001605/52570732-68fdb280-2e57-11e9-8d23-606417a65916.png)

![files](https://user-images.githubusercontent.com/35001605/52572986-72d5e480-2e5c-11e9-9187-323fc0273f8c.PNG)

![train_files](https://user-images.githubusercontent.com/35001605/52573708-0e1b8980-2e5e-11e9-95a0-f6dc1b5c93d4.PNG)

![densenet2](https://user-images.githubusercontent.com/35001605/53281173-8e4bc400-3767-11e9-8938-c27190f6c0e0.png)

![resnet2](https://user-images.githubusercontent.com/35001605/53281175-9146b480-3767-11e9-89e0-fa8ea75f925c.png)

![vgg2](https://user-images.githubusercontent.com/35001605/53281180-9a378600-3767-11e9-95d4-d0b6fe751d3a.png)

![densenet2_down_2](https://user-images.githubusercontent.com/35001605/53281568-1c2aad80-376e-11e9-9130-8fc38042be4a.png)

![resnet2_down_2](https://user-images.githubusercontent.com/35001605/53281608-cf93a200-376e-11e9-9679-758ce8d59dc1.png)

![sh run2](https://user-images.githubusercontent.com/35001605/53282876-68cbb400-3781-11e9-86e7-e94a0173320b.PNG)
![sh run1](https://user-images.githubusercontent.com/35001605/53282877-68cbb400-3781-11e9-9b31-aa53b4b5f280.PNG)

![training set inside](https://user-images.githubusercontent.com/35001605/53282752-b2b39a80-377f-11e9-8083-dee42a57303e.PNG)

![validation set inside](https://user-images.githubusercontent.com/35001605/53282754-b515f480-377f-11e9-8495-aa84ad7c3ae1.PNG)

![val_after_cleanining](https://user-images.githubusercontent.com/35001605/53282873-5baec500-3781-11e9-8c12-befceb819075.PNG)
